routes/index.js

```diff
@@ -254,42 +243,48 @@ module.exports = function (passport) {
     if (!req.body) {
       return res.sendStatus(400);
     }
-    var BagdeID = req.params.badge;
-    dbAPI.modifyUser(BagdeID, { confirmation: true }).then(function (result) {
+    var badgeID = req.params.badge;
+    dbAPI.modifyUser(badgeID, { confirmation: true }).then(function (result) {
       if (result == undefined) {
         console.log("Error");
       }
       else {
         console.log("confirmation changed successfully");
       }
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is cofirmed successfully');
+    req.flash('fade_out', '3000');
+    res.redirect('/userManagement/' + badgeID);
   });
 
 
   router.post('/userManagement/deletePrivilege/:badge/:station', checkAuth, function (req, res) {
     if (!req.body) {
       return res.sendStatus(400);
     }
-    var BagdeID = req.params.badge;
+    var badgeID = req.params.badge;
     var StationID = req.params.station;
-    dbAPI.removePrivileges(BagdeID, StationID).then(function (result) {
+    dbAPI.removePrivileges(badgeID, StationID).then(function (result) {
       console.log(result);
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is revoked from this machine');
+    req.flash('fade_out', '3000');
+    res.redirect('/userManagement/' + badgeID);
   });
 
 
   router.post('/userManagement/grantPrivilege/:badge/:station', checkAuth, function (req, res) {
     if (!req.body) {
       return res.sendStatus(400);
     }
-    var BagdeID = req.params.badge;
+    var badgeID = req.params.badge;
     var StationID = req.params.station;
-    dbAPI.grantPrivileges(BagdeID, StationID).then(function (result) {
+    dbAPI.grantPrivileges(badgeID, StationID).then(function (result) {
       console.log(result);
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is granted to this machine');
```

Comment:
Change this to "The user has been successfully granted access to the selected station"